scripts/publish-site.sh

```diff
@@ -11,6 +11,11 @@ yarn
 echo "=== Copying built Gatsby to website."
 gatsby-dev --scan-once --quiet
 
+echo "=== find graphql packages"
+
+find node_modules -name graphql
+echo "=== END find graphql packages"
```

Comment:
it should ensure a single instance of it given a fresh install but you know...npm

Maybe the best approach is actually to expose graphql to plugins like babel does for babel.types. like pass in the `graphql' object